### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Prajeshpuri**

### CODE REVIEW

**Comment by ankushgarg1998** [comment]
🔥

**Comment by Prajeshpuri** [comment]
does it need any more changes ?

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ So this is a minimalist result website I'm making for our University students, i
   - **College** *(Ex: MSIT, MAIT, BVCOE...)*, 
   - **Shift** *(Ex: Morning, Evening, Both)*, 
   - **Batch** *(Ex: 2016-20, 2017-21...)*, 
+  - **Course** *(Ex: BTech and BCA)*,
```
Comment (ankushgarg1998) [inline README.md:26]: Add BBA as well please. 😛

File: README.md [hunk 2]
```diff
@@ -36,7 +37,7 @@ So this is a minimalist result website I'm making for our University students, i
 ## Planned Features
 - An overall section for institutes, that shows a ranklist which includes students from all the colleges of a particular branch.
 - Ranks in the profile section.
-- Integration of BCA and BBA courses.
+- Integration of BBA courses.
```
Comment (ankushgarg1998) [inline README.md]: Done 😎. Remove this please.

**Pull request closed without merging**